### PR TITLE
Changes cursor position to end of inserted image syntax, implements #2079 

### PR DESCRIPTION
**Pull request by rsynnest**
Changes cursor position to end of text if media manager is in 'keep open' mode for adding multiple images (Issue #2079).

### CODE REVIEW

File: lib/scripts/media.js [hunk 1]
```diff
@@ -937,15 +938,18 @@ var dw_mediamanager = {
...
     var alignleft = '';
     var alignright = '';
     if (align !== '1') {
         alignleft = align === '2' ? '' : ' ';
         alignright = align === '4' ? '' : ' ';
     }
-
-    insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + '|', '}}', '');
+    if (keepopen) {
+        insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + '|}}', '', '');
```
Comment (phy25) [inline lib/scripts/media.js]: I think after the tag we can add `\n\n` for better composition. That is:
```js
insertTags(edid, '{{' + alignleft + mediaid + opts + alignright + "|}}\n\n", '', '');
```

Comment (splitbrain) [reply lib/scripts/media.js]: I would disagree. I often use the feature to insert multiple images into one article but not necessarily one after each other. More like "one here, on there and another there..." having it insert new paragraphs would be counter productive then.

Comment (phy25) [reply lib/scripts/media.js]: @splitbrain is right. I ignored this use case before.

**Comment by Klap-in** [comment]
I would like that if the media manager includes just one image (or the latest if you include a couple of images), that it puts the cursor in the just included syntax (the behaviour before this change).

My proposal is to check if the cursor is already in a syntax (that is checking if `}}` exists after the cursor), and put the next inserted image after it.

**Comment by splitbrain** [comment]
@Klap-in suggestion makes much more sense to me. @rsynnest could you adjust your changes accordingly?

**Comment by rsynnest** [comment]
sure thing, will push an update soon

**Comment by splitbrain** [comment]
good work. Thanks a lot @rsynnest :+1:

**Pull request merged**